File: be/booking/src/main/java/com/example/booking/model/dto/BookingDto.java
package com.example.booking.model.dto;

import lombok.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class BookingDto {
    private String phonenumber;
    private Double latitude;
    private Double longtitude;
    private String typeCar;
    private String state;
    private String paymentMethod;
    private Float price;
}

File: fe/User/app/src/main/java/com/example/user/data/dto/BookingDto.kt
package com.example.user.data.dto

import com.example.user.utils.PaymentMethod
import com.example.user.utils.TypeCar
import com.google.gson.annotations.SerializedName

data class BookingDto(
    @SerializedName("dropoffLocation")
    val destination: LatLong,
    val paymentMethod: String,
    @SerializedName("phonenumber")
    var phoneNumber: String = "",
    @SerializedName("pickupLocation")
    val origin: LatLong,
    val price: Double,
    val typeCar: String,
    var username: String = ""
)